test/simple_ng/simple_ng.c

```diff
@@ -0,0 +1,318 @@
+/** \file
+ * \brief Example code for Simple Open EtherCAT master
+ *
+ * Usage: simple_ng IFNAME1
+ * IFNAME1 is the NIC interface name, e.g. 'eth0'
+ *
+ * This is a minimal test.
+ */
+
+#include "ethercat.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+typedef struct {
+    ecx_contextt    context;
+    char *          iface;
+    uint8           group;
+    int             roundtrip_time;
+
+    /* Used by the context */
+    uint8           map[4096];
```

Comment:
Set the map to 0's? 

Or clear the entire filedbus map? Otherwiase you'll start with heap values, that can be confusing for users.      
memset(&fieldbus, 0, sizeof(fieldbus));

Reply:
Thank you for taking the time to review.
As I have no experience with SOEM, I strictly followed what already done in [ec_master.c](https://github.com/OpenEtherCATsociety/SOEM/blob/master/test/intime/ec_master/ec_master.c#L168), where `IOmap` is not initialized.
Not a big deal though, I will add the whole struct clearing as suggested: I'm already doing it for `ecx_contextt`, so it just needs to be extended to the whole `Fieldbus` struct.

Reply:
Sure , global variabels get 0’ed by default. Stack variabels doesn’t, unless they’re static

Reply:
> Sure , global variabels get 0’ed by default.

You're right, never known about that.